### ISSUE TITLE
Install breakage with latest setuptools?

### ISSUE THREAD

**Issue by meeas** [riptideio/pymodbus#117]
I think there may have been something in the latest setuptools that is choking on something in pymodbus/**init**.py.  It doesn't appear to install through pip nor by download with manual install.  Here is the error:

```
Traceback (most recent call last):
  File "setup.py", line 26, in <module>
    from pymodbus import __version__, __author__
  File "/private/var/root/pymodbus/pymodbus/__init__.py", line 37
    True, False = (1 == 1), (0 == 1)
       ^
SyntaxError: can't assign to keyword
```

Commenting out the try/except section which makes up the last four lines of that file fixes the issue allowing pymodbus to install cleanly.

**Comment by barik** [comment]
Depending on the version of Python you are using, you may wish to try `pip install pymodbus3`.

**Comment by geoffe** [comment]
Ah, thanks.  Although I found a more active project minimalmodbus which may suit just as well.

**Comment by Tutunkommon** [comment]
Install is still broken.  Looked at minimalmodbus but it doesn't support TCP.  Hoping pymodbus3 covers everything!

**Comment by dhoomakethu** [comment]
fixed with pymodbus >= 1.3.0
